src/Angor/Shared/WalletOperations.cs

```diff
@@ -156,6 +237,11 @@ public List<UtxoDataWithPath> FindOutputsForTransaction(long sendAmountat, Accou
                      .OrderBy(o => o.utxo.blockIndex)
                      .ThenByDescending(o => o.utxo.value))
         {
+            if (unconfirmedInfo.AccountPendingSpent.Any(p => p.outpoint.ToString() == utxoData.utxo.outpoint.ToString()))
+            {
+                continue;
+            }
```

Comment:
Why not add it as a Where in the Utxos..SelectMany

Reply:
good idea done